README.md

```diff
@@ -54,186 +52,24 @@ client
-
-const welcomeImage = fs.readFileSync(path.join(__dirname, "welcome.png"));
+Refer to the [`examples`](examples) folder for the source code of this and other advanced examples.
 
-client
-  .send({
-    category: "test",
-    custom_variables: {
-      hello: "world",
-      year: 2022,
-      anticipated: true,
-    },
-    from: sender,
-    to: [{ email: RECIPIENT_EMAIL }],
-    subject: "Hello from Mailtrap!",
-    html: `
-    <!doctype html>
-    <html>
-      <head>
-        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-      </head>
-      <body style="font-family: sans-serif;">
-        <div style="display: block; margin: auto; max-width: 600px;" class="main">
-          <h1 style="font-size: 18px; font-weight: bold; margin-top: 20px">Congrats for sending test email with Mailtrap!</h1>
-          <p>Inspect it using the tabs you see above and learn how this email can be improved.</p>
-          <img alt="Inspect with Tabs" src="cid:welcome.png" style="width: 100%;">
-          <p>Now send your email using our fake SMTP server and integration of your choice!</p>
-          <p>Good luck! Hope it works.</p>
-        </div>
-        <!-- Example of invalid for email html/css, will be detected by Mailtrap: -->
-        <style>
-          .main { background-color: white; }
-          a:hover { border-left-width: 1em; min-height: 2em; }
-        </style>
-      </body>
-    </html>
-  `,
-    attachments: [
-      {
-        filename: "welcome.png",
-        content_id: "welcome.png",
-        disposition: "inline",
-        content: welcomeImage,
-      },
-    ],
-  })
-  .then(console.log)
-  .catch(console.error)
-```
+### General API
 
-### Mail from template
+ - [List User & Invite account accesses](examples/general/account-accesses.ts)
+ - [Remove account access](examples/general/accounts.ts)
+ - [Permissions](examples/general/permissions.ts)
 
-```ts
-import { MailtrapClient } from "mailtrap"
+### Email testing API
 
-/**
- * For this example to work, you need to set up a sending domain,
- * and obtain a token that is authorized to send from the domain.
- */
-
-const TOKEN = "<YOUR-TOKEN-HERE>";
-const SENDER_EMAIL = "<SENDER@YOURDOMAIN.COM>";
-const RECIPIENT_EMAIL = "<RECIPIENT@EMAIL.COM>";
-
-const client = new MailtrapClient({ token: TOKEN });
-
-client
-  .send({
-    from: { name: "Mailtrap Test", email: SENDER_EMAIL },
-    to: [{ email: RECIPIENT_EMAIL }],
-    template_uuid: "813e39db-c74a-4830-b037-0e6ba8b1fe88",
-    template_variables: {
-      user_name: "John Doe",
-    },
-  })
-  .then(console.log)
-  .catch(console.error);
-```
+ - [Attachments](examples/testing/attachments.ts)
+ - [Inboxes](examples/testing/inboxes.ts)
+ - [Messages](examples/testing/messages.ts)
+ - [Projects](examples/testing/projects.ts)
 
 ### Nodemailer Transport
 
-> NOTE: [Nodemailer](https://www.npmjs.com/package/nodemailer) is needed as a dependency.
-
-```sh
```

Comment:
I would probably leave installation steps here, and just replace code example with a link 🙏 
- Reading installation steps in Readme would be more comfortable

Reply:
Agree, updating